### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by tgurbich**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,61 +2,337 @@
...
 ```
 
 > **Warning:**
 > Please provide pipeline parameters via the CLI or Nextflow `-params-file` option. Custom config files including those
 > provided by the `-c` Nextflow option can be used to provide any configuration _**except for parameters**_;
 > see [docs](https://nf-co.re/usage/configuration#custom-configuration-files).
 
+<a name="test"></a>
+## Test
+To run the pipeline using  a test dataset, execute the following command:
```
Comment (mberacochea) [inline README.md:197]:
```suggestion
To run the pipeline using a test dataset, execute the following command:
```

File: README.md [hunk 2]
```diff
@@ -2,61 +2,337 @@
...
+- Install [Nextflow version >=21.10](https://www.nextflow.io/docs/latest/getstarted.html#installation)
+- Install [Singularity](https://github.com/apptainer/singularity/blob/master/INSTALL.md)
+- Install [Docker](https://docs.docker.com/engine/install/)
+
+Although it's possible to run the pipeline on a personal computer, due to the compute requirements, we encourage users to run it on HPC clusters. Any HPC scheduler supported by [Nextflow](https://www.nextflow.io/) is compatible; however, our team primarily uses [Slurm](https://slurm.schedmd.com/) and [IBM LSF](https://www.ibm.com/docs/en/spectrum-lsf) for the EBI HPC cluster, so those are the profiles we ship with the pipeline.
+
+<a name="reference-databases"></a>
+### Reference databases
+
+The pipeline needs reference databases in order to work, they take roughtly 110G.
```
Comment (mberacochea) [inline README.md:71]:
```suggestion
The pipeline needs reference databases in order to work, they take roughly 110G.
```

File: README.md [hunk 3]
```diff
@@ -2,61 +2,337 @@
...
+
+#### Data sources
+Below is an explanation of how each field in column 3 and 9 of the final GFF file is populated. In most cases, information is taken as is from the reporting tool's output.
+
+| Feature (column 3)    | Attribute Name (column 9)                                               | Reporting Tool  | Description                                                                                                                                                                                                 |
+|-----------------------|-------------------------------------------------------------------------|-----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ncRNA                 | all                                                                     | cmscan + Rfam   | ncRNA annotation (excluding tRNA)                                                                                                                                                                           |
```
Comment (mberacochea) [inline README.md:271]: Does 'all' indicate that the entire row in the GFF is populated by this tool?

If that is so, maybe "as is" or similar? Or all* and have an explanation in the table footer?

**Review by mberacochea — APPROVED** [review]
This looks very nice :). I just left you 2 comments, and wil send you my edits for the `product` schema in a minute

File: README.md (hunk 3 above)
Comment (tgurbich) [reply README.md:271]: It means all attributes in column 9...

**Pull request merged**